doc/index.rst

```diff
@@ -1,12 +1,78 @@
 .. _framework:
 
 =====================
-Core functionalities
+ROS2 Control API
 =====================
 
 `GitHub Repository <https://github.com/ros-controls/ros2_control>`_
 
-Features
-=========
+Controller Interface
```

Comment:
Would it make sense to add the docs for each package in that package?

This would result in including/indexing "sub-index" files from packages into this existing file. Then the files from which doxygen generated docs should be inluded there (in a separate file or "sub-index" file.)

This is how we did it with controllers. (for example, see L50 in [this file](https://github.com/ros-controls/ros2_controllers/blob/master/doc/controllers_index.rst)

Reply:
I don't quite see how that would be different from the current solution except pushing the doxygen directive to a reStructedText file that would then be included by doc/index.rst. If you're trying to get around the autogenerate problem, I think this would still have that issue.